packages/phone-number-privacy/src/config.ts

```diff
@@ -38,6 +40,7 @@ if (DEV_MODE) {
       unverifiedQueryMax: 2,
       additionalVerifiedQueryMax: 30,
       queryPerTransaction: 2,
+      minDollarBalance: new BigNumber(100000000000000000),
```

Comment:
Can we have a constant like cUSDToWei (whatever the smallest unit is called) and derive this value like:
`minDollarBalance: new BigNumber(.1 * cUSDToWei)`

This way it's self documenting and less error prone.

Reply:
good idea -- i'm not sure what we call the smallest unit.  

in our migrationconfig we seem just to include all the zeros every time, but this idea seems better